packages/trader/src/Stores/Modules/Portfolio/Helpers/details.js

```diff
@@ -37,13 +37,14 @@ export const getDurationUnitValue = obj_duration => {
 
 export const isEndTime = duration => duration % 1 !== 0;
 
+export const unit_map = {
+    d: { name_plural: localize('days'), name_singular: localize('day') },
```

Comment:
localize won't work in a const, it needs to be a function

Reply:
or it needs to be `<Localize` to re-render once languages have loaded

Reply:
Sure. I'll make the changes needed 👍